src/main/kotlin/org/hyacinthbots/allium/extensions/Modrinth.kt

```diff
@@ -81,102 +83,82 @@ class Modrinth : Extension() {
-                                index.toString() // leave this as else this doesn't work (please I don't want to count up manually)
-                                var loaders = ArrayList<String>()
-                                for (loader in vers_hit.asJsonObject.getAsJsonArray("loaders")) {
-                                    loaders.add(loader.asString)
-                                }
-                                m.addAll(loaders)
-                            }
-                            var strLoaders = ""
-                            for (entry in m) {
-                                strLoaders += entry + "\n"
-                            }
-                            strLoaders.dropLast(2)
-                            page {
-                                this.title = hit["title"].asString
-                                this.url = "https://modrinth.com/project/${hit["slug"].asString}"
-                                thumbnail {
-                                    this.url = hit["icon_url"].asString
-                                }
-                                this.description = hit["description"].asString
-                                field("Latest Version", true) { hit["latest_version"].asString }
-                                field(
-                                    "Client/Server Side",
-                                    true
-                                ) { "Client: ${hit["client_side"].asString}\nServer: ${hit["server_side"].asString}" }
-                                field("Downloads", true) { hit["downloads"].asString }
-                                field("Author", true) { hit["author"].asString }
-                                field(
-                                    "Last Update",
-                                    true
-                                ) { "<t:${Instant.parse(hit["date_modified"].asString).epochSeconds}>" }
-                                field("License", true) { hit["license"].asString }
-                                field("Loaders", true) { strLoaders }
-                                footer {
-                                    this.text = "Modrinth | ${hit["author"].asString}"
+                    } else {
+                        respondingPaginator {
+                            for ((i, _) in hits.withIndex()) {
+                                val hit: JsonObject = hits.get(i).asJsonObject
+
+                                val strLoaders = getProjectLoaders(hit)
+
+                                page {
+                                    embedContents(hit, strLoaders)
                                 }
                             }
-                        }
-                        timeoutSeconds = 60
-                        locale = Locale.ENGLISH
-                    }.send()
+                            timeoutSeconds = 300
+                            locale = Locale.ENGLISH
+                        }.send()
```

Comment:
Also why the longer timeout?

Reply:
Better accessibility for users, gives them more time to read. Can be reduced back down to 2 or 3 minutes as oppose to 5

Reply:
I think 3 minutes would be enough

Reply:
Done